src/background/environment.js

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export async function initializeEnvironment() {
+  // Issue #57: Disable login saving for extension pages.
+  const origin = browser.extension.getURL("").slice(0, -1);
+  const savingEnabled = await browser.experiments.logins
+    .getLoginSavingEnabled(origin);
+  if (savingEnabled) {
```

Comment:
Checking whether saving is enabled to ensure we keep it off. The user could enable login save globally or manually delete the exception from the list in options. Kind of edge casey, but bleh.

Reply:
It seems unlikely that users are going to mess with this manually. That said, the browser APIs are available from extension pages, so we could just run this check when the management page is loaded (or, to get kind of extreme about the "user messes things up" edge case, we could run this check on page load and whenever the page visibility API's `visibilitychange` event fires).